### ISSUE TITLE
Nations Upkeep Error

### ISSUE THREAD

**Issue by GershonR** [Arckenver/Nations#73]
```
25.12 12:00:05 [Server] Server thread/ERROR [Sponge]: Could not pass MoveEntityEvent$Impl to Plugin{id=nations, name=Nations, version=2.1, description=A towny-like worldguard-like zone managment plugin., url=https://github.com/Arckenver/Nations, authors=[Arckenver], source=/mods/Nations-2.1.jar}
25.12 12:00:05 [Server] INFO java.util.ConcurrentModificationException
25.12 12:00:05 [Server] INFO at java.util.Hashtable$Enumerator.next(Hashtable.java:1378) ~[?:1.8.0_112]
25.12 12:00:05 [Server] INFO at com.arckenver.nations.DataHandler.getNation(DataHandler.java:127) ~[DataHandler.class:2.1]
25.12 12:00:05 [Server] INFO at com.arckenver.nations.listener.PlayerMoveListener.onPlayerMove(PlayerMoveListener.java:35) ~[PlayerMoveListener.class:2.1]
25.12 12:00:05 [Server] INFO at org.spongepowered.common.event.listener.MoveEntityEventListener_PlayerMoveListener_onPlayerMove283.handle(Unknown Source) ~[?:?]
25.12 12:00:05 [Server] INFO at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95) ~[RegisteredListener.class:1.10.2-2123-6.0.0-BETA-1902]
25.12 12:00:05 [Server] INFO at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:357) [SpongeModEventManager.class:1.10.2-2123-6.0.0-BETA-1902]
25.12 12:00:05 [Server] INFO at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:398) [SpongeModEventManager.class:1.10.2-2123-6.0.0-BETA-1902]
25.12 12:00:05 [Server] INFO at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:370) [SpongeModEventManager.class:1.10.2-2123-6.0.0-BETA-1902]
25.12 12:00:05 [Server] INFO at org.spongepowered.common.SpongeImpl.postEvent(SpongeImpl.java:144) [SpongeImpl.class:1.10.2-2123-6.0.0-BETA-1902]
25.12 12:00:05 [Server] INFO at net.minecraft.network.NetHandlerPlayServer.redirect$throwMoveEvent$zgp000(NetHandlerPlayServer.java:461) [me.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.network.NetHandlerPlayServer.func_147347_a(NetHandlerPlayServer.java:406) [me.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.network.play.client.CPacketPlayer.func_148833_a(SourceFile:126) [iw.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.network.play.client.CPacketPlayer$Position.func_148833_a(SourceFile:57) [iw$a.class:?]
25.12 12:00:05 [Server] INFO at org.spongepowered.common.network.PacketUtil.onProcessPacket(PacketUtil.java:135) [PacketUtil.class:1.10.2-2123-6.0.0-BETA-1902]
25.12 12:00:05 [Server] INFO at net.minecraft.network.PacketThreadUtil$1.redirect$onProcessPacket$zjc000(SourceFile:39) [fl$1.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [fl$1.class:?]
25.12 12:00:05 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_112]
25.12 12:00:05 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_112]
25.12 12:00:05 [Server] INFO at net.minecraft.util.Util.func_181617_a(SourceFile:45) [h.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
25.12 12:00:05 [Server] INFO at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
25.12 12:00:05 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_112]
25.12 12:00:05 [Server] pool-3-thread-17/INFO Nation {NATION} could not pay its upkeep and fell into ruins
```

**Comment by TheoKah** [comment]
I see how to fix this. I'll do it asap

**Comment by TheoKah** [comment]
This was fixed in PR #77
